src/ripple/rpc/handlers/Submit.cpp

```diff
@@ -47,7 +47,7 @@ Json::Value doSubmit (RPC::Context& context)
         return rpcError (rpcINVALID_PARAMS);
 
     Serializer                  sTrans (ret.first);
-    SerializerIterator          sitTrans (sTrans);
+    SerialIter          sitTrans (sTrans);
```

Comment:
I agree, I find no utility in left justifying the second or third column of automatic variable declarations like this (with a tiny number of exceptions, such as tabular figures):
```
int                          i
std::string                  s
std::shared_ptr<Transaction> spt;
```

But if you're asking why it is no longer left justified it is because it was done using a global search and replace.